packages/yoga/src/Heading/web/StyledHeading.jsx

```diff
@@ -1,6 +1,11 @@
 import styled, { css } from 'styled-components';
+import Box from '../../Box';
 
-const StyledHeading = styled.header`
+const StyledHeading = styled(Box).attrs(props => ({
+  as: 'header',
+  bg: props.theme.yoga.components.heading,
+  ...props,
+}))`
```

Comment:
The Box component should not be used inside the Yoga by definition, if we add it here ti will bring to the component some extra props as `margins`, for instance.
Could we add the background color support for the yoga system approach, WDYT?

```suggestion
import { backgroundColor } from '@gympass/yoga-system';

const StyledHeading = styled.header`
  ${backgroundColor}
```

Reply:
It makes sense @ericcleao, I made the change you suggested. Thanks!